MediaSession.cpp

```diff
@@ -638,17 +641,13 @@ CDMi_RESULT MediaKeySession::Decrypt(
     *f_ppbOpaqueClearContent = (uint8_t *)payloadData;
 #endif
 
-/* PRv3.3 support */
-#ifdef PR_3_3
-        if( f_pcbOpaqueClearContent != NULL )
-          {
-              *f_pcbOpaqueClearContent = 0;
-          }
-          if( f_ppbOpaqueClearContent != NULL )
-          {
-              *f_ppbOpaqueClearContent = NULL;
-          }
-#endif
+ErrorExit:
+    if (DRM_FAILED(dr)) {
+      const DRM_CHAR* description;
+      DRM_ERR_GetErrorNameFromCode(dr, &description);
+      printf("playready error on Decrypt: %s\n", description);
```

Comment:
The code above is using fprintf(stderr, "error string") on error paths, so it would make sense to do the same here

Reply:
Thanks for identifying this